pkgs/applications/science/molecular-dynamics/lammps/default.nix

```diff
@@ -1,54 +1,63 @@
 { stdenv, fetchFromGitHub
 , libpng, gzip, fftw, openblas
+, cmake, pkgconfig, python2
 , mpi ? null
 }:
-let packages = [
-     "asphere" "body" "class2" "colloid" "compress" "coreshell"
-     "dipole" "granular" "kspace" "manybody" "mc" "misc" "molecule"
-     "opt" "peri" "qeq" "replica" "rigid" "shock" "snap" "srd" "user-reaxc"
-    ];
-    lammps_includes = "-DLAMMPS_EXCEPTIONS -DLAMMPS_GZIP -DLAMMPS_MEMALIGN=64";
-    withMPI = (mpi != null);
+let
+  withMPI = (mpi != null);
+
+  packages = [
```

Comment:
Could you perhaps make this configurable from `callPackage`? This could be nice if someone doesn't want _all_ of the bells and whistles.